cudax/include/cuda/experimental/__stf/internal/constants.cuh

```diff
@@ -35,11 +35,13 @@ namespace cuda::experimental::stf
  */
 enum class access_mode : unsigned int
 {
-  none    = 0,
-  read    = 1,
-  write   = 2,
-  rw      = 3, // READ + WRITE
-  relaxed = 4, /* operator ? */
+  none           = 0,
+  read           = 1,
+  write          = 2,
+  rw             = 3, // READ + WRITE
+  relaxed        = 4, /* operator ? */
+  reduce_no_init = 8,
+  reduce_do_init = 16,
```

Comment:
```suggestion
  reduce_init = 8,
  reduce = 16,
```

I mean the implicit operation is "reduce" and there's a special thing "reduce initialize".

Reply:
I guess the default should be the reduce, and the less common case would be reduce_no_init, so i'll do that instead